### ISSUE TITLE
HELP ME!!! AttributeError: 'LSTM' object has no attribute '_flat_weights_names'

### ISSUE THREAD

**Issue by Ajefrez** [facebookresearch/demucs#73]
Una vez termina el proceso sale esto. y no encuentro los archivos que separó y no sé si los separó o no....

(base) C:\Users\Jef\demucs-master>python -m demucs.separate --dl -n demucs "C:\Users\Jef\Music\Descargas deezer\Alvinsch - Ser Normal.mp3"
Downloading pre-trained model weights, this could take a while...
100%|█████████████████████████████████████████████████████████████████████████████| 2.59G/2.59G [10:01<00:00, 4.31MB/s]
Traceback (most recent call last):
  File "C:\Users\Jef\Anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\Jef\Anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Jef\demucs-master\demucs\separate.py", line 155, in <module>
    main()
  File "C:\Users\Jef\demucs-master\demucs\separate.py", line 126, in main
    model = th.load(model_path).to(args.device)
  File "C:\Users\Jef\Anaconda3\lib\site-packages\torch\nn\modules\module.py", line 425, in to
    return self._apply(convert)
  File "C:\Users\Jef\Anaconda3\lib\site-packages\torch\nn\modules\module.py", line 201, in _apply
    module._apply(fn)
  File "C:\Users\Jef\Anaconda3\lib\site-packages\torch\nn\modules\module.py", line 201, in _apply
    module._apply(fn)
  File "C:\Users\Jef\Anaconda3\lib\site-packages\torch\nn\modules\rnn.py", line 137, in _apply
    self._flat_weights = [getattr(self, weight) for weight in self._flat_weights_names]
  File "C:\Users\Jef\Anaconda3\lib\site-packages\torch\nn\modules\module.py", line 576, in __getattr__
    type(self).__name__, name))
AttributeError: 'LSTM' object has no attribute '_flat_weights_names'

**Comment by MJCastroM** [comment]
I have exactly the same problem. Someone??

Traceback (most recent call last):
  File "/home/marcos/anaconda3/envs/demucs/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/marcos/anaconda3/envs/demucs/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/marcos/Descargas/demucs-master/demucs/separate.py", line 155, in <module>
    main()
  File "/home/marcos/Descargas/demucs-master/demucs/separate.py", line 126, in main
    model = th.load(model_path).to(args.device)
  File "/home/marcos/anaconda3/envs/demucs/lib/python3.7/site-packages/torch/nn/modules/module.py", line 425, in to
    return self._apply(convert)
  File "/home/marcos/anaconda3/envs/demucs/lib/python3.7/site-packages/torch/nn/modules/module.py", line 201, in _apply
    module._apply(fn)
  File "/home/marcos/anaconda3/envs/demucs/lib/python3.7/site-packages/torch/nn/modules/module.py", line 201, in _apply
    module._apply(fn)
  File "/home/marcos/anaconda3/envs/demucs/lib/python3.7/site-packages/torch/nn/modules/rnn.py", line 137, in _apply
    self._flat_weights = [getattr(self, weight) for weight in self._flat_weights_names]
  File "/home/marcos/anaconda3/envs/demucs/lib/python3.7/site-packages/torch/nn/modules/module.py", line 576, in __getattr__
    type(self).__name__, name))
AttributeError: 'LSTM' object has no attribute '_flat_weights_names'

**Comment by adefossez** [comment]
Try forcing the install of pytorch 1.3.1,
```
conda uninstall pytorch
conda install pytotch==1.3.1 -c pytorch
```
There seem to be an incompatibility with pytorch-1.4.0. I'll try to fix that but for now the easiest is to stay with 1.3.1.

**Comment by MJCastroM** [comment]
It works! Thanks!

**Comment by adefossez** [comment]
I fixed this issue with in the latest commit. You will have to redownload the checkpoints though as I had no choice but to change the serialization format.

**Comment by edufierro** [comment]
Hi! 

I'm facing this error too and trying to avoid pinning torch to <1.4. 

Is this release still in master or was it released already? Testing to read models with 1.5.0 and models trained with 1.3.  

Thanks!
